### DISCUSSION TITLE
PV 计数问题

### DISCUSSION THREAD

**Discussion by inkss** [ArtalkJS/Artalk#111] [Ideas]
观察到页面未设置 PV 元素时，程序没有计数。

是否修改成不管页面是否存在计数元素，只要页面上存在着评论框，就加加呢？或者添加一个全局开关？

**Comment by qwqcode** [comment]
你可以 `<span style="display: none;"></span>`

**Reply by qwqcode** [reply]
之后改成无论元素是否存在，都计数

**Reply by inkss** [reply]
哈哈哈，主要是想到其他的评论系统逻辑上是那样做的

**Reply by qwqcode** [reply]
OK

**Reply by xiamuguizhi** [reply]
@qwqcode 好吧，我更新了最新版本 `Artalk (v2.1.2/3c58913)`  然后之前的pv统计浏览量 全重新开始统计了 .....

之前的PV统计 没有保存的吗？

**Reply by qwqcode** [reply]
😅 之前的在数据库的 PVs 数据表里，因为我把这个和 pages 数据表合并了。你可以使用 SQL 命令导过去
